src/App.js

```diff
@@ -79,6 +85,9 @@ export default class App extends Component {
         if (next.parent !== prev.parent) return currentTheme.separationCousins;
       });
       this.setState({ currentTheme });
+      this.setState({
+        currentCustomTheme: currentTheme,
```

Comment:
mmmh, I think what you want to do here is to set both `currentTheme` and `currentCustomTheme`